CustomerContactBook/CustomerContactBook/Controllers/CustomerGroupsController.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using CustomerContactBook.Models;
+
+namespace CustomerContactBook.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class CustomerGroupsController : ControllerBase
+    {
+        private readonly CustomerGroupContext _context;
+
+        public CustomerGroupsController(CustomerGroupContext context)
+        {
+            _context = context;
+        }
+
+        // GET: api/CustomerGroups
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<CustomerGroup>>> GetGroups()
+        {
+            return await _context.Groups.ToListAsync();
+        }
+
+        // GET: api/CustomerGroups/5
+        [HttpGet("{id}")]
+        public async Task<ActionResult<CustomerGroup>> GetCustomerGroup(int id)
+        {
+            var customerGroup = await _context.Groups.FindAsync(id);
+
+            if (customerGroup == null)
+            {
+                return NotFound();
+            }
+
+            return customerGroup;
+        }
+
+        // PUT: api/CustomerGroups/5
+        // To protect from overposting attacks, see https://go.microsoft.com/fwlink/?linkid=2123754
+        [HttpPut("{id}")]
+        public async Task<IActionResult> PutCustomerGroup(int id, CustomerGroup customerGroup)
```

Comment:
`CustomerGroup` is class that directly mapped into database table, for controllers please use another one (``CustomerGroup` is class that directly mapped into database table, please create another one `CustomerGroupDto`), like described in [the link](https://go.microsoft.com/fwlink/?linkid=2123754) from comment of this method.

So there will be:
```
public class CustomerGroup // this class maps to database
{
    public long Id { get; set; }
    public string Name { get; set; }
}

public class CustomerGroupDto // this class is parameter of controller methods
{
    public long Id { get; set; }
    public string Name { get; set; }
}

public class CustomerGroupCreateDto // this class is parameter of controller methods
{
    public string Name { get; set; }
}
```